data/introTexts.json

```diff
@@ -43,4 +43,5 @@
     ["good soldiers", "follow orders"],
     ["not the original game", "this is a complete remake"],
     ["DrPatrick", "was here"]
```

Comment:
You missed a comma here. Add a comma after this line.

Reply:
Fixed